metadata-ingestion/src/datahub/ingestion/source/sql/athena.py

```diff
@@ -303,6 +303,11 @@ class AthenaConfig(SQLCommonConfig):
         print_warning=True,
     )
 
+    convert_simple_field_paths_to_v1 = pydantic.Field(
+        default=False,
+        description="Convert simple field paths to datahub field path v1 format. Simple column paths are those that do not contain any nested fields.",
```

Comment:
```suggestion
        description="Convert simple field paths to DataHub field path v1 format. Simple column paths are those that do not contain any nested fields.",
```

Isn't the simple paths the v1 already? 🤔  I understand simple path is v1, and complex is v2 😅 
Maybe `emit_schema_fieldpaths_as_v1` or something like that....